src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -21,16 +21,22 @@
 class SearchConceptsQueryBuilderTest {
 
   private SnapshotBuilderDomainOption createTestSnapshotBuilderDomainOption(
-      String category, int id) {
-    return new SnapshotBuilderDomainOption().category(category).id(id);
+      String name, int id, String occurrenceTable, String columnName) {
+    return (SnapshotBuilderDomainOption)
+        new SnapshotBuilderDomainOption()
+            .name(name)
+            .id(id)
+            .tableName(occurrenceTable)
+            .columnName(columnName);
```

Comment:
For this use case I would split into a declaration and then setter calls to avoid the need for a cast
```suggestion
    var option = new SnapshotBuilderDomainOption()
        option.name(name)
            .id(id)
            .tableName(occurrenceTable)
            .columnName(columnName);
    return option;
```